website/docs/users-sources/sources/directory-sync/active-directory/index.md

```diff
@@ -62,6 +62,7 @@ Additional settings that might need to be adjusted based on the setup of your do
 - User object filter: Which objects should be considered users. For Active Directory set it to `(&(objectClass=user)(!(objectClass=computer)))` to exclude Computer accounts.
 - Group object filter: Which objects should be considered groups.
 - Group membership field: Which user field saves the group membership
+- Lookup using user attribute: Lookup group memberships from a user object attribute instead of a group attribute (`memberOf` instead of `member`). It can be useful for looking up nested group memberships, for which you'd want to use `memberOf:1.2.840.113556.1.4.1941:` as the group membership field, to tell Active Directory to follow DNs.
```

Comment:
```suggestion
- Look up using a user attribute: Look up group memberships based on a user object attribute instead of a group attribute (`memberOf` instead of `member`). It can be useful for looking up nested group memberships, for which you'd want to use `memberOf:1.2.840.113556.1.4.1941:` as the group membership field, to tell Active Directory to follow DNs.
```

Reply:
Thanks @krejcar25 !